tests/config/test_functional_config_loading.py

```diff
@@ -86,14 +86,14 @@ def test_functional_config_loading(
     # 'rstrip()' applied, so we can have a final newline in the expected test file
     assert expected_output.rstrip() == out.rstrip(), msg
     assert sorted(expected_loaded_configuration.keys()) == sorted(
-        runner.linter.config.__dict__.keys()
+        runner.linter.namespace.__dict__.keys()
```

Comment:
Yes, but this also makes it so that we actually test all config options. (See OP).
By using `linter.config` we are only testing options from the `PyLinter` class